Recrutify/Recrutify.Host/Startup.cs

```diff
@@ -43,19 +44,33 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddControllers()
                 .AddFluentValidation();
+
             services.AddValidators();
 
             services.Configure<RouteOptions>(options => options.LowercaseUrls = true);
 
-            services.AddControllers().AddOData(opt => opt.Filter().Expand().Select().OrderBy().SetMaxTop(100).AddRouteComponents("odata", GetEdmModel()));
+            services.AddApiVersioning(options =>
+            {
+                options.DefaultApiVersion = new ApiVersion(1, 0);
+                options.AssumeDefaultVersionWhenUnspecified = true;
+            });
+
+            services.AddOData().EnableApiVersioning();
+            services.AddODataApiExplorer(
+                options =>
+                {
+                    options.GroupNameFormat = "'v'VVV";
```

Comment:
try to remove - and check if it works without it

Reply:
I checked it right away - it doesn't work